README.md

```diff
@@ -75,15 +75,16 @@ As more reliable or faster versions of the exploit are developed, the images are
 |-------------------|--------|------------------------|---------------|
 | 1.0 (1994-09-22)  | SCPH-1000 | **Yes** | [20210421](images/freepsxboot-unirom-fastload-20210421-bios-1.0.mcd) |
 | 1.1 (1995-01-22)  | SCPH-3000 | **Yes** | [20210421](images/freepsxboot-unirom-fastload-20210421-bios-1.1.mcd) |
+| 2.0 (1995-05-07)  | SCPH-1001 | **Yes; see note below** | [20210421](images/freepsxboot-unirom-fastload-20210421-bios-2.0.mcd) |
```

Comment:
Please update the note below with SCPH-1001, as well as the 3rd bullet point in the "Usage" section.